lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -86,6 +86,19 @@ class AwsCompileEventBridgeEvents {
           required: ['inputTemplate'],
           additionalProperties: false,
         },
+        retryPolicy: {
+          type: 'object',
+          properties: {
+            maximumEventAge: { type: 'integer' },
+            maximumRetryAttempts: { type: 'integer' },
```

Comment:
Are there minimum and maximum values for that property allowed by AWS? If yes, let's add them here to avoid potential CloudFormation validation issues and catch those invalid values earlier